tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Can you also save the figure as a PNG or whatever? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
You mean you want an example of doing `savefig` or similar?  Is the purpose to show the reader how to save a figure, or to have something saved for CI or other technical reasons?

Reply:
just to show the reader how to save a figure. We went to all that effort to make a figure, let's publish it! or put it on a poster or something.

Reply:
added